src/app/app.page/app.page.component.html

```diff
@@ -495,6 +495,32 @@ <h4 class="card-title text-left" i18n="@@newLiveStreamCardTitle">
                                                         </button>
 
 
+                                                        <!-- Stop button IP Camera & Stream Source-->
+                                                        <button *ngIf="(row.type== 'streamSource' || row.type=='ipCamera') && row.status=='broadcasting'"
+                                                                (click)="stopStreams(row.streamId)"
+                                                                class="btn btn-simple btn-magnify btn-info btn-xs big-icons"
+                                                                type="button" data-toggle="tooltip" i18n-title
+                                                                title="Stop Stream" >
+                                                            <span class="btn-label" style="float: right;">
+                                                                <i class="ti-control-pause"></i>
+                                                            </span>
+
+                                                            <ng-container i18n="@@tableSettings"></ng-container>
+                                                        </button>
+
+                                                        <!-- Start button IP Camera & Stream Source -->
+                                                        <button *ngIf="(row.type=='streamSource' || row.type=='ipCamera') && row.status!='broadcasting'"
+                                                                (click)="startStreams(row.streamId)"
+                                                                class="btn btn-simple btn-magnify btn-info btn-xs big-icons"
+                                                                type="button" data-toggle="tooltip" i18n-title
+                                                                title="Start Stream" >
+                                                            <span class="btn-label" style="float: right;">
+                                                                <i class="ti-control-play"></i>
```

Comment:
Please don't use play button for starting stream.  Alternative option is context menu with three vertical dots. You can put delete, embed copy code, settings menu items to this context menu as well. 

Context Menu
<img width="247" alt="Screen Shot 2019-12-09 at 08 50 50" src="https://user-images.githubusercontent.com/3456251/70410743-23068080-1a61-11ea-8b1a-2126615afc21.png">

Vertical dots
https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQC7dlRcMVEhtzqK55UP4HSzDuBrg0ETvpmSZnWvh5s6iXhguaM